### ISSUE TITLE
One-shot example in chatgpt vs open source comparison is wrong

### ISSUE THREAD

**Issue by kvaughan-ct** [microsoft/guidance#23]
There is an issue with this notebook:

https://github.com/microsoft/guidance/blob/main/notebooks/chatgpt_vs_open_source_on_harder_tasks.ipynb

The one-shot example for the simple equation is incorrect:

> For example, if the solution is x=2, say SOLUTION = [3].

I believe the numbers there are meant to align to each other. Not sure if it would impact the evaluation results, but worth being accurate.

**Comment by slundberg** [comment]
Thanks! That does look like a typo. We can update that

**Comment by marcotcr** [comment]
Ugh, thanks for catching this.
Fixed in aa7ed050a0ba9894edcd025639a878bc51c97fac, and updated the blog post with new numbers:
```
Frequency of correct answers:
ChatGPT:  1.0
Vicuna:  0.9285714285714286
MPT:  0.23076923076923078
```
